packages/grafana-ui/package.json

```diff
@@ -78,6 +78,7 @@
     "@types/react-custom-scrollbars": "4.0.5",
     "@types/react-test-renderer": "16.9.0",
     "@types/react-transition-group": "2.0.16",
+    "@types/react-virtualized-auto-sizer": "1.0.0",
```

Comment:
Let's remove it from grafana-ui, as it's used in withFullSizeStory.tsx which is not used anywhere (let's remove it too :D)